vpr/src/place/place.cpp

```diff
@@ -833,6 +1039,23 @@ void try_place(const t_placer_opts& placer_opts,
     VTR_LOG("\tSwaps rejected: %*d (%4.1f %%)\n", num_swap_print_digits, num_swap_rejected, 100 * reject_rate);
     VTR_LOG("\tSwaps aborted : %*d (%4.1f %%)\n", num_swap_print_digits, num_swap_aborted, 100 * abort_rate);
 
+    float moves, accepted, rejected, aborted;
```

Comment:
Helper function:  print_placement_move_stats ()

Reply:
Overall this is already a really long routine, and its getting longer.  Look for ways to create helper functions and remove duplicate code to shorten.

Reply:
Done by adding two helper function to print the overall swaps stats and the detailed move type stats.